### PR TITLE
Sequence PCTP acceptance

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

File: app/controllers/health/appointments_controller.rb [hunk 1]
```diff
@@ -17,9 +17,14 @@ def index
       set_hpc_patient
       # set_patient if @patient.blank?
       a_t = Health::Appointment.arel_table
-      @appointments = @patient.appointments.order(appointment_time: :desc)
+      @appointments = if @patent.present?
```
Comment (eanders) [inline app/controllers/health/appointments_controller.rb]: Maybe you meant @patient?
```suggestion
      @appointments = if @patient.present?
```

File: app/controllers/health/appointments_controller.rb [hunk 2]
```diff
@@ -36,10 +41,15 @@ def upcoming
       else
         end_date = start_date + 2.weeks
       end
-      @appointments = @patient.appointments.
-        limited.
-        where(appointment_time: (start_date..end_date)).
-        order(appointment_time: :asc)
+      @appointments = if @patent.present?
```
Comment (eanders) [inline app/controllers/health/appointments_controller.rb]:
```suggestion
      @appointments = if @patient.present?
```

**Review by eanders — CHANGES_REQUESTED** [review]
I think there were some typos in there.  I also wonder if those would be more readable as:
```
@appointments = Health::Appointment.none
@appointments = @patient.scope.... if @patient.present?
```

**Comment by awisema** [comment]
Thanks... An interesting risk to the fact that `@non_existent_attribute` is not an error...

I tried the proposed alternative, but decided I found it less clear, so I kept the if functions.

**Review by eanders — APPROVED** [review]

**Pull request merged**